server/db/actions/Dog.js

```diff
@@ -3,7 +3,7 @@ import Dog from "../models/Dog";
 import Log from "../models/Log";
 import User from "../models/User";
 
-export async function getDogs(filter = {}) {
+export async function getDogs(filter = {}, fields = null) {
```

Comment:
What does `fields` do?

Reply:
filters just for those values to be returned just like graphQL. so i use this for getting the data for dogs dropdown since I only need the id and the name and i don't need any other value.